### PR TITLE
Update README.md with a link to Colin's blog post

### PR DESCRIPTION
**Pull request by brittanybelle**
Colin recently wrote an outstanding blog post that would be especially helpful to new users of oysteR - let's promote it directly in the README!

cc @bhamail / @DarthHater / @csgillespie

### CODE REVIEW

**Comment by csgillespie** [comment]
@brittanybelle Made a few tweaks to the PR
 * Remember, you need to update the `Rmd` file not the `md` file ;)
 * Moved the logo. Avoids the need for links outside of the domain. 
 * Resized the logo

File: README.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
...
-================
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-<p align="center">
-
-<img src="https://github.com/sonatype-nexus-community/oysteR/blob/master/docs/images/OysteR_LabeledInHex.png" width="350"/>
-
-</p>
+# oysteR <a href='https://tidyverse.tidyverse.org'><img src='man/figures/logo.png' align="right" height="138.5" /></a>
```
Comment (brittanybelle) [inline README.md:4]: I'm thinking maybe this was copypasta from a different readme? But I do like the suggestion of making the img ref relative.

```suggestion
# oysteR <a href='https://github.com/sonatype-nexus-community/oysteR'><img src='docs/images/OysteR_LabeledInHex.png' align="right" height="138.5" /></a>
```

File: README.Rmd [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   chunk_output_type: console
 ---
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
-<p align="center">
-    <img src="https://github.com/sonatype-nexus-community/oysteR/blob/master/docs/images/OysteR_LabeledInHex.png" width="350"/>
-</p>
+
+
+# oysteR <a href='https://tidyverse.tidyverse.org'><img src='man/figures/logo.png' align="right" height="138.5" /></a>
```
Comment (brittanybelle) [inline README.Rmd:10]: I'm thinking maybe this was copypasta from a different repo?

```suggestion
# oysteR <a href='https://github.com/sonatype-nexus-community/oysteR'><img src='docs/images/OysteR_LabeledInHex.png' align="right" height="138.5" /></a>
```

**Comment by brittanybelle** [comment]
Thanks a bunch, @csgillespie ! Haven't returned to the R space in a few weeks so I totally forgot about .Rmd :) Thanks for the other details you added, too.

I noticed the logo you updated was pointing to tidyverse and left a couple comments including a suggested change, would appreciate your eyes on those as a sanity check when you have a moment (Jeffry is out of office for the next couple of weeks). Thank you!

File: README.md (hunk 1 above)
Comment (csgillespie) [reply README.md]: Good spot @brittanybelle - this is why we have code review ;)

**Comment by csgillespie** [comment]
@brittanybelle 

- I've updated the URL to point to this repo
- I've moved the logo from `docs/` to `man/` This means that the logo will now "come with the package". It's a semi-standard R thing
- Updated the Rmd and regenerated the `.md` file

**Comment by bhamail** [comment]
FWIW, I just reran the CI build and it was fine. Looks like an intermittent issue.
I'll merge this in now the CI is happy.

**Comment by brittanybelle** [comment]
Ack, thanks for merging this @bhamail ! I totally lost the plot over the long weekend.

Thanks a bunch for polishing this up, @csgillespie !

**Pull request merged**